src/libsync/progressdispatcher.h

```diff
@@ -91,6 +91,19 @@ class OWNCLOUDSYNC_EXPORT ProgressInfo : public QObject
      */
     void adjustTotalsForFile(const SyncFileItem &item);
 
+    /**
+     * Update totals for item.
+     * adjustTotalsForFile is called during the treewalk phase to collect
+     * the initial total size and file count.
+     * updateTotalsForFile is called at most once per item during propagation
+     * to adjust them when new information has become available.
+     *
+     * Example: With delta-sync, the actual size of the download will only
+     * be known during propagation - this function adjusts the total size
+     * to account for it.
+     */
```

Comment:
for later: This might need even more documentation: item._size must be the old size when this is called, different from `newSize` which isn't obvious to me.